index.html

```diff
@@ -2701,317 +2728,11 @@ <h2>
-                        <var>request</var>.<a>[[\details]]</a>.<a>modifiers</a>
-                        to <var>details</var>.<a>modifiers</a>.
-                        </li>
-                        <li>Set
-                        <var>request</var>.<a>[[\serializedModifierData]]</a>
-                        to <var>serializedModifierData</var>.
-                        </li>
-                      </ol>
-                    </li>
-                    <li>If <var>request</var>.<a>[[\options]]</a>.<a data-lt=
-                    "PaymentOptions.requestShipping">requestShipping</a> is
-                    true, and
-                    <var>request</var>.<a>[[\details]]</a>.<a>shippingOptions</a>
-                    is empty, then the developer has signified that there are
-                    no valid shipping options for the currently-chosen shipping
-                    address (given by <var>request</var>'s <a data-lt=
-                    "PaymentRequest.shippingAddress">shippingAddress</a>). In
-                    this case, the user agent SHOULD display an error
-                    indicating this, and MAY indicate that the currently-chosen
-                    shipping address is invalid in some way. The user agent
-                    SHOULD use the <a data-lt=
-                    "PaymentDetailsUpdate.error">error</a> member of
-                    <var>details</var>, if it is present, to give more
-                    information about why there are no valid shipping options
-                    for that address.
-                    </li>
-                  </ol>
-                </li>
-              </ol>
-            </li>
-            <li>In either case, run the following steps, after either the upon
-            rejection or upon fulfillment steps have concluded:
-              <ol>
-                <li>Set <var>request</var>.<a>[[\updating]]</a> to false.
-                </li>
-                <li>The <a>user agent</a> SHOULD update the user interface
-                based on any changed values in <var>request</var>. The user
-                agent SHOULD re-enable user interface elements that might have
-                been disabled in the steps above if appropriate.
-                </li>
-              </ol>
-            </li>
-          </ol>
-          <p>
-            If any of the above steps say to <dfn>abort the update</dfn> with
-            an exception <var>exception</var>, then:
-          </p>
-          <ol>
-            <li>Abort the current user interaction and close down any remaining
-            user interface.
-            </li>
-            <li>Set <var>request</var>.<a>[[\state]]</a> to "<a>closed</a>".
-            </li>
-            <li>Reject the promise <var>request</var>.<a>[[\acceptPromise]]</a>
-            with <var>exception</var>.
-            </li>
-            <li>Abort the algorithm.
+            <li>Return from the method and, <a>in parallel</a>, run the
```

Comment:
This shouldn't be done in parallel, it turns out; the algorithm just waits on promises, which should be done on the main thread (like it is done in JS).